From 49ddc64e5beb762132f2691be04b1b9be094a57d Mon Sep 17 00:00:00 2001
Message-Id: <49ddc64e5beb762132f2691be04b1b9be094a57d.1387382496.git.minovotn@redhat.com>
In-Reply-To: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
References: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
From: Nigel Croxon <ncroxon@redhat.com>
Date: Thu, 14 Nov 2013 22:53:18 +0100
Subject: [PATCH 42/46] spec: add buildrequires for librdmacm-devel for rdma
 support

RH-Author: Nigel Croxon <ncroxon@redhat.com>
Message-id: <1384469598-13137-43-git-send-email-ncroxon@redhat.com>
Patchwork-id: 55723
O-Subject: [RHEL7.0 PATCH 42/42] spec: add buildrequires for librdmacm-devel for rdma support
Bugzilla: 1011720
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Applied patch from Amit Shah <amit.shah@redhat.com>

Bugzilla: 1011720
https://bugzilla.redhat.com/show_bug.cgi?id=1011720
---
 redhat/qemu-kvm.spec.template |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 redhat/qemu-kvm.spec.template | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/redhat/qemu-kvm.spec.template b/redhat/qemu-kvm.spec.template
index 8b1fc28..9220681 100644
--- a/redhat/qemu-kvm.spec.template
+++ b/redhat/qemu-kvm.spec.template
@@ -18,6 +18,10 @@
     %global have_usbredir 0
 %endif
 
+%ifnarch s390 s390x
+    %global have_librdma 1
+%endif
+
 %ifnarch x86_64
     %global build_only_sub 1
     %global debug_package %{nil}
@@ -201,6 +205,10 @@ BuildRequires: pixman-devel
 # Documentation requirement
 BuildRequires: perl-podlators
 BuildRequires: texinfo
+# For rdma
+%if 0%{?have_librdma:1}
+BuildRequires: librdmacm-devel
+%endif
 
 %if 0%{!?build_only_sub:1}
 Requires: qemu-img = %{epoch}:%{version}-%{release}
-- 
1.7.11.7

